R/valley.R

```diff
@@ -38,15 +37,9 @@ get_dem <- function(bb, dem_source = "STAC", stac_endpoint = NULL,
                     stac_collection = NULL, crs = NULL) {
   bbox <- as_bbox(bb)
   if (dem_source == "STAC") {
-    if (is.null(stac_endpoint) && is.null(stac_collection)) {
```

Comment:
I have moved these defaults downstream to facilitate some testing of the cache - and I think they make more sense there.